### DISCUSSION TITLE
connect from second mc client error

### DISCUSSION THREAD

**Discussion by dazx2k** [minio/minio#12498] [Q&A]
Hi there

I cant connect to minio server from from second client.

Client 1
My first client on windows working fine.
mc.exe alias set minio https://ip address:9000 password1 password2
mc mb myminio/backup

Client 2
My second client on linux failed
./mc alias set gw https://ip address:9000 password1 password2

mc: <ERROR> Unable to initialize new alias from the provided credentials. Get "h                                                                             ttps://ip address:9000/probe-bucket-sign-3gm539vjze1z/?location=": x509: can                                                                             not validate certificate for ip address because it doesn't contain any IP SANs. 

Any idea.

Thanks

**Comment by mazataza** [comment]
Try with mc -- insecure
